### ISSUE TITLE
[Question]: what is "Storage" in Cypress accessibility checker task?

### ISSUE THREAD

**Issue by Jeremias-Bertoldi** [IBMa/equal-access#1489]
### Project

cypress accessibility checker

### Description

Hi there, great tool you have made!
My question is: what is "Storage" in this tool, because I'm getting this error in my workflow, and I don't understand very well what happen:

cy.task('accessibilityChecker') failed with the following error:
> Failed to read a named property from 'Storage': access is denied for this document.

This fails in this task: accessibilityChecker, {task: sendResultsToReporter, data: <document>}
Looks like the tool is not able to generate a report, is this a problem related to NPM or node?

Hop you can help me.

Node version: v14.19.1
NPM version: 9.5.0
Cypress version: ^12.0.2
cypress-accessibility-checker: ^3.1.50
cypress-cucumber-preprocessor: ^4.3.0

**Comment by tombrunet** [comment]
There aren't any references to "Storage" in our codebase. That said, I think I ran into something similar while working on #1500. That PR will clean up a lot of the reporting-related code. For Cypress, it will move a lot of the logic from the Command side to the Task side. It also pulls some of the Cypress fixes that we put out as a 4.x release candidate into the 3.x releases. You might try `[4.0.0-rc.0](https://www.npmjs.com/package/cypress-accessibility-checker/v/4.0.0-rc.0)` and see if that resolves it for you for now.

We have some summer vacations delaying that from moving along, but hoping to get that out maybe late next week or the week after. Will leave this open until we can get that code released for you to verify.

**Comment by Jeremias-Bertoldi** [comment]
Hi tom, thank you for answer. I've changed the version but another error appeared:
**Error:    CypressError: `cy.task()` must only be invoked from the spec file or support file.**
Does it require another kind of implementation?
Hope you can help me.

**Comment by tombrunet** [comment]
Sorry about the version hopping, but `3.1.51` was just released with a lot of the report cleanup. Can you please check that?

**Comment by Jeremias-Bertoldi** [comment]
Hi Tom, got this error:
<img width="857" alt="Captura de pantalla 2023-07-05 a la(s) 14 10 38" src="https://github.com/IBMa/equal-access/assets/100160945/dfec7ec0-6369-4015-b064-d4eb46907d78">
Is this your side or mine, the required node version is 18 right? I'm currently using  v20.3.1 (npm v9.6.7)

**Comment by tombrunet** [comment]
Looks like our side. Our tests completed without issues, but that does look like our code. Looking to see if something didn't work in deployment.

**Comment by tombrunet** [comment]
@Jeremias-Bertoldi 3.1.53 should fix that bit. There was a .gitignore file in the src tree and npm decided to not deploy those files as a result.

**Comment by Jeremias-Bertoldi** [comment]
@tombrunet Got this error, I don't know if it's a bad configuration of my cypress environment or anything else. 
<img width="1440" alt="Captura de pantalla 2023-07-06 a la(s) 09 52 06" src="https://github.com/IBMa/equal-access/assets/100160945/9359ae9a-d884-4867-92ea-a677ab450e97">

what do you think?
The tool module-deps ( https://www.npmjs.com/package/module-deps) tries to create a json file of all my dependency tree and cannot parse the ACEngineManager file.
Hope you can help me.

**Comment by tombrunet** [comment]
@Jeremias-Bertoldi Sorry - not sure how I didn't see your comment. If I understand, it's not just running our tool that's causing you trouble, but rather running module-deps while also having cypress-accessibility-checker that's failing?

**Comment by Jeremias-Bertoldi** [comment]
Hi @tombrunet you can close this ticket, I've imported cypress-accessibility-checker outside of package-modules and it worked. I hope in the future it would be fixed, but until now it works

**Comment by MHoov** [comment]
Closing per Jeremias' comment above.
